CONTRIBUTING.md

```diff
@@ -3,35 +3,39 @@
 ## Setting Up Development Environment
 
 ### 1 Fork and clone the repository.
+https://help.github.com/articles/fork-a-repo/
 
-### 2 Install Vagrant.
-https://www.vagrantup.com/downloads.html
+### 2 Install Docker.
+https://docs.docker.com/install/
 
-### 3 Install VirtualBox
-https://www.virtualbox.org/wiki/Downloads
+### 3 Build the Docker Container
+Build the container
+```
+docker-compose build
+```
 
-### 4 Capture the powers of vagrant
-  * In the repo dir: <code>vagrant up</code> (Safely ignore: 'dpkg-preconfigure: unable to re-open stdin: No such file or directory')
-  * If changes have been made since running vagrant up: <code>vagrant provision</code>
-  
-  * Start the rails server. There are two ways to do this, depending on your familiarity running from within the vagrant shell:
+### 4 Run the Docker Container
 
-    1. A local `rake` wrapper that allows direct execution on the machine.
-        * Run `rake vagrant:shell[command]`<sup>1</sup>
-        * To start the rails server using the rake wrapper use: `rake vagrant:shell['rails s -b 0.0.0.0']`.         
-        * Navigate to `localhost:3000`
-    2. Using `vagrant ssh` to gain access directly to the machine.
-        * To login to the machine: <code>vagrant ssh</code>
-        * `cd  /vagrant/` to navigate to the refuge repo.
-        * To start the rails server use: `rails s -b 0.0.0.0`. 
-        * Navigate to `localhost:3000`
+You can now run and shutdown the app with:
+```
+docker-compose up
+docker-compose down
+```
 
-  [1] You can run any command locally using `rake vagrant:shell[]` and it will be executed in the repo root of the vagrant machine. You can try `rake vagrant:shell['pwd']` and see it will print the directory that the repo is in on the vagrant machine!
+It will be available at localhost:3000
 
-### 5 Optional tasks:
-run <code>rake db:fix_accents</code> to clean up encoding problems in the safe2pee data. (Use <code>rake db:fix_accents[dry_run]</code> to preview the changes.)
+### 5 Run the Tests
+```
+docker-compose run -e "RAILS_ENV=test" web rake db:test:prepare spec cucumber
+```
 
-### 6 Assets
+### 6 Optional tasks:
```

Comment:
@tkwidmer do we need this anymore? I feel if we do, it might not belong in this doc

Reply:
I'm not sure if we still need this. The seed has been reduced to a small sample set rather than a larger set that it used to be. So we may not need this.

Reply:
I'm going to delete since in my past 3.5 years of working on this I've never done this. It might just be confusing to a newcomer